src/components/SubscriptionsPage.vue

```diff
@@ -67,18 +67,20 @@
 
     <ModalComponent v-if="showEditGroupModal" @close="showEditGroupModal = false">
         <h2>{{ selectedGroup.groupName }}</h2>
-        <div class="flex flex-col mt-3 mb-2">
-            <div v-for="subscription in subscriptions" :key="subscription.name">
-                <div class="flex justify-between">
-                    <span>{{ subscription.name }}</span>
-                    <input
-                        type="checkbox"
-                        class="checkbox"
-                        :checked="selectedGroup.channels.includes(subscription.url.substr(-11))"
-                        @change="checkedChange(subscription)"
-                    />
+        <div style="overflow-y: scroll; height: 300px">
```

Comment:
Please use the appropriate unocss/tailwind classes instead.

For example, it would be: `overflow-y-scroll` and a suitable height.

Reply:
Ok sure

Reply:
done